go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/giantswarm/k8sclient/v2
 go 1.14
 
 require (
-	github.com/giantswarm/apiextensions v0.4.17-0.20200723160042-89aed92d1080
+	github.com/giantswarm/apiextensions/v2 v2.1.0
```

Comment:
I believe `v0.4.17-0.20200723160042-89aed92d1080` was the working branch which was released as `v2` so this should be equivalent.

Reply:
Yes, that's correct.